src/Stratis.Bitcoin.Features.Consensus/ProvenBlockHeaders/IProvenBlockHeaderStore.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using NBitcoin;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Consensus.ProvenBlockHeaders
+{
+    /// <summary>
+    /// Cache layer for <see cref="ProvenBlockHeader"/>s.
+    /// </summary>
+    public interface IProvenBlockHeaderStore : IProvenBlockHeaderProvider, IDisposable
```

Comment:
IDisposable is already in base interface

Reply:
Ok, also removed on `IProvenBlockHeaderRepository`.